azurerm/internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -85,25 +85,76 @@ func resourceDatabricksWorkspace() *pluginsdk.Resource {
 				Default:  false,
 			},
 
+			"public_network_access_enabled": {
+				Type:     pluginsdk.TypeBool,
+				ForceNew: true,
+				Optional: true,
+				Default:  true,
+			},
+
+			"require_network_security_group_rules": {
```

Comment:
should we be consistant with the tense?
```suggestion
			"network_security_group_rules_required": {
```

Reply:
Fixed.